Entity/CategoryManager.php

```diff
@@ -97,6 +97,32 @@ public function getSubCategoriesPager($categoryId, $page = 1, $limit = 25, $crit
     }
 
     /**
+     * @param CategoryInterface $category
+     *
+     * @return CategoryInterface
+     */
+    public function getRootCategoryWithChildren($category)
```

Comment:
Ok… better have tests than final methods I guess. The solution could also be to not mock it, or mock an interface.